src/data/api.js

```diff
@@ -4,15 +4,52 @@ import { ExamAction } from '../constants';
 
 const BASE_API_URL = '/api/edx_proctoring/v1/proctored_exam/attempt';
 
+async function fetchActiveAttempt() {
+  const activeAttemptUrl = new URL(`${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`);
+  const activeAttemptResponse = await getAuthenticatedHttpClient().get(activeAttemptUrl.href);
+  return activeAttemptResponse.data.attempt;
+}
+
 export async function fetchExamAttemptsData(courseId, sequenceId) {
-  const url = new URL(
-    `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
-  );
-  if (sequenceId) {
-    url.searchParams.append('content_id', sequenceId);
+  let data;
+  if (!getConfig().EXAMS_BASE_URL) {
+    const url = new URL(
+      `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
+    );
+    if (sequenceId) {
+      url.searchParams.append('content_id', sequenceId);
+    }
+    url.searchParams.append('is_learning_mfe', true);
+    const urlResponse = await getAuthenticatedHttpClient().get(url.href);
+    data = urlResponse.data;
+  } else {
+    const examUrl = new URL(`${getConfig().EXAMS_BASE_URL}/api/v1/student/exam/attempt/course_id/${courseId}/content_id/${sequenceId}`);
+    const examResponse = await getAuthenticatedHttpClient().get(examUrl.href);
+    data = examResponse.data;
+
+    const attemptData = await fetchActiveAttempt();
+    data.active_attempt = attemptData;
+  }
+  return data;
+}
+
+//
+export async function fetchLatestAttempt(courseId) {
+  let data;
+  if (!getConfig().EXAMS_BASE_URL) {
+    const url = new URL(
+      `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
+    );
+    url.searchParams.append('is_learning_mfe', true);
```

Comment:
🤔 wonder if we can drop setting this everywhere at this point, there's no case where this isn't true anymore.

Reply:
Is Open edX only using the learning MFE for exams now?

Reply:
yeah, by extension of the non-mfe courseware views being deprecated the old exams views are as well.

Reply:
Can create a follow up ticket to change the backend behavior

Reply:
@alangsto I already tossed this in https://2u-internal.atlassian.net/browse/MST-1830